src/irmin/backend.ml

```diff
@@ -94,6 +94,7 @@ module type S = sig
     val contents_t : t -> read Contents.t
     val node_t : t -> read Node.t
     val commit_t : t -> read Commit.t
+    val config : t -> Conf.t
```

Comment:
I don't particularly love the `_t` suffix but what do you think about `config_t` for consistency?

Reply:
Normally I would agree, but in this case the `_t` functions all return backend stores, so I think calling it `config_t` is a bit misleading.